src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx

```diff
@@ -49,7 +49,7 @@ const AddressWithAutofill = props => {
       [...REQUIRED_ADDRESS_FIELDS, 'street2'].forEach(field => {
         const fieldName = `root_${name}_${field}`;
         const inputElement = document.getElementById(fieldName);
-        if (inputElement) formData[field] = inputElement.value;
+        formData[field] = inputElement?.value;
```

Comment:
There is no way to test the `if` conditional here. Let's gracefully use the optional chain to handle the edge case of the element being undefined.